modal/modal.css

```diff
@@ -0,0 +1,143 @@
+* {
+    font-family: 'Nunito Sans', sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.component-title {
+    font-size: 1.8rem;
+    font-weight: bold;
+    margin-bottom: 1.5rem;
+}
+
+.component-description {
+    font-size: 1.1rem;
+    margin-bottom: 1.5rem;
+}
+
+.btn {
+    text-align: center;
+}
+
+.live-demo-button {
+    padding: 20px 20px;
```

Comment:
Suggestion: padding:20px , will affect the same.

Reply:
> Suggestion: padding:20px , will affect the same.

my bad.will keep that in mind .thanks for pointing out